### PR TITLE
Fix billing tests

### PR DESCRIPTION
**Pull request by thgreasi**
Change-Type: patch
See: https://www.flowdock.com/app/rulemotion/resin-frontend/threads/b7HXSaRvyA5a315JCMnluTy7F57

### CODE REVIEW

File: tests/integration/models/billing.spec.coffee [hunk 1]
```diff
@@ -24,35 +24,35 @@ describe 'Billing Model', ->
 		describe 'resin.models.billing.getPlan()', ->
 			it 'should return a free tier billing plan object', ->
 				promise = resin.models.billing.getPlan()
```
Comment (pimterry) [inline tests/integration/models/billing.spec.coffee]: You can get rid of the `promise =` now too

File: tests/integration/models/billing.spec.coffee [hunk 2]
```diff
@@ -142,52 +142,41 @@ describe 'Billing Model', ->
...
 						title: 'TestDev'
 						name: 'TestDev plan'
 						code: 'testdev'
 						tier: 'testdev'
 						addOns: []
-						intervalUnit: 'months'
-						intervalLength: '1'
-					})
+						# intervalUnit: 'months'
+						# intervalLength: '1',
```
Comment (pimterry) [inline tests/integration/models/billing.spec.coffee]: Let's just remove these entirely.

File: tests/integration/models/billing.spec.coffee [hunk 3]
```diff
@@ -142,52 +142,41 @@ describe 'Billing Model', ->
...
-								quantity: '1'
-							}
-						]
-						totalCostCents: '0'
-					})
-					m.chai.expect(plan).to.have.property('support').that.deep.equals({
-						title: 'Standard'
-						name: 'Standard support'
-					})
+					# m.chai.expect(plan).to.have.property('description').that.is.a('string')
```
Comment (pimterry) [inline tests/integration/models/billing.spec.coffee]: ditto

File: tests/integration/models/billing.spec.coffee (hunk 1 above)
Comment (thgreasi) [reply tests/integration/models/billing.spec.coffee]: Oh, missed that.

**Pull request merged**